var/spack/repos/builtin/packages/flit/package.py

```diff
@@ -3,27 +3,32 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-
 from spack import *
 
-
```

Comment:
Removing this line is causing Flake8 to fail

Reply:
Done